java/src/main/java/com/microsoft/graph/bulk/impl/BulkGremlinObjectMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.graph.bulk.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.microsoft.graph.bulk.impl.model.GremlinEdge;
+import com.microsoft.graph.bulk.impl.model.GremlinVertex;
+
+public final class BulkGremlinObjectMapper {
+    private BulkGremlinObjectMapper() {
+        throw new IllegalStateException("Utility class, should not be constructed");
+    }
+
+    /**
+     * Defines an ObjectMapper that has the Serialization Modules for both GremlinVertex and GremlinEdge classes
+     */
+    public static ObjectMapper getBulkGremlinObjectMapper() {
+        var mapper = new ObjectMapper();
```

Comment:
I see we are using `var` everywhere, which was introduced in java 10, which means this sample cannot support Java 8. 

We cannot use `var` because azure-cosmos Java SDKs still support java 8. So if a cosmos Java SDK customer is using Java 8, this sample won't work for them. We should use strict types in the samples.